src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -178,9 +178,10 @@ const ColonyHome = ({
     // Use ROOT_DOMAIN if filtered domain id equals 0
     mergePayload({
       colonyAddress,
-      domainId: filteredDomainId || ROOT_DOMAIN,
+      colonyName,
```

Comment:
Why is the `colonyName` needed?

Reply:
I'll return this from the `CreateTaskMutation` instead (in the saga for the purpose of redirecting the user to the task after it's created) 👍